### PR TITLE
Search RBAC clarifications

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/27132

### CODE REVIEW

File: access_control/rbac_implement_rhacm.adoc [hunk 1]
```diff
@@ -233,9 +233,23 @@ To view the observability metrics for a managed cluster, you must have `view` ac
 * Access managed cluster metrics.
 +
 Users are denied access to managed cluster metrics, if they are not assigned to the `view` role for the managed cluster on the hub cluster.
++
+Run the following command to add view access for an user: 
++
+----
+oc auth can-i create ManagedClusterView -n <managedClusterName> --as=<user>
+----
```
Comment (Anxhela21) [inline access_control/rbac_implement_rhacm.adoc:239]: the following command is not to add view access for a user but to verify if the user has the authority to create a managedClusterView in the managed cluster namespace

Comment (Anxhela21) [reply access_control/rbac_implement_rhacm.adoc:239]: If you want to add instructions to create a role for the user to create a ManagedClusterView in the managed cluster namespace as <user> see following:

1. First the administrator (kubeadmin) needs to create a role for creating managedclusterview in the managed cluster namespace with the following command:
`oc create role create-managedclusterview --verb=create --resource=managedclusterviews -n <managedClusterName>`

2. Then the administrator needs to apply/bind this role to the user by creating a role-binding:
`oc create rolebinding user-create-managedclusterview-binding --role=create-managedclusterview --user=<user>  -n <managedClusterName>`

File: access_control/rbac_implement_rhacm.adoc [hunk 2]
```diff
@@ -233,9 +233,23 @@ To view the observability metrics for a managed cluster, you must have `view` ac
 * Access managed cluster metrics.
 +
 Users are denied access to managed cluster metrics, if they are not assigned to the `view` role for the managed cluster on the hub cluster.
```
Comment (Anxhela21) [inline access_control/rbac_implement_rhacm.adoc:235]: I'm not sure metrics is the right word here. Maybe we can use resources instead to be more general?

File: access_control/rbac_implement_rhacm.adoc (hunk 1 above)
Comment (dockerymick) [reply access_control/rbac_implement_rhacm.adoc:239]: Updated

**Comment by dockerymick** [comment]
Thanks for your review @Anxhela21 can you re-review when you have a chance?

File: access_control/rbac_implement_rhacm.adoc [hunk 3]
```diff
@@ -232,11 +232,38 @@ To view the observability metrics for a managed cluster, you must have `view` ac
 
 * Access managed cluster metrics.
 +
-Users are denied access to managed cluster metrics, if they are not assigned to the `view` role for the managed cluster on the hub cluster.
-* Search for resources.
+Users are denied access to managed cluster metrics, if they are not assigned to the `view` role for the managed cluster on the hub cluster. Run the following command to verify if an user has the authority to create a `managedClusterView` role in the managed cluster namespace:
```
Comment (oafischer) [inline access_control/rbac_implement_rhacm.adoc]:
```suggestion
Users are denied access to managed cluster metrics, if they are not assigned to the `view` role for the managed cluster on the hub cluster. Run the following command to verify if a user has the authority to create a `managedClusterView` role in the managed cluster namespace:
```

**Review by oafischer — APPROVED** [review]
lgtm, 1 suggestion

**Comment by dockerymick** [comment]
@oafischer thanks so much for your review, suggestion, and approval! I made some more updates. @Anxhela21 may you also re-review?

File: access_control/rbac_implement_rhacm.adoc [hunk 4]
```diff
@@ -232,11 +232,38 @@ To view the observability metrics for a managed cluster, you must have `view` ac
...
+Users are denied access to managed cluster metrics, if they are not assigned to the `view` role for the managed cluster on the hub cluster. Run the following command to verify if a user has the authority to create a `managedClusterView` role in the managed cluster namespace:
++
+----
+oc auth can-i create ManagedClusterView -n <managedClusterName> --as=<user>
+----
++
+As a cluster administrator, create a `managedClusterView` role in the managed cluster namespace. Run the following command:
++
+----
+oc create role create-managedclusterview --verb=create --resource=managedclusterviews -n <managedClusterView>
```
Comment (Anxhela21) [inline access_control/rbac_implement_rhacm.adoc]:
```suggestion
oc create role create-managedclusterview --verb=create --resource=managedclusterviews -n <managedClusterName>
```

File: access_control/rbac_implement_rhacm.adoc (hunk 1 above)
Comment (Anxhela21) [reply access_control/rbac_implement_rhacm.adoc:239]: corrected above should be in namespace `<managedClusterName>` NOT `<managedClusterView>`

File: access_control/rbac_implement_rhacm.adoc [hunk 5]
```diff
@@ -232,11 +232,38 @@ To view the observability metrics for a managed cluster, you must have `view` ac
...
+----
++
+Then apply and bind the role to a user by creating a role bind. Run the following command:
++
+----
+oc create rolebinding user-create-managedclusterview-binding --role=create-managedclusterview --user=<user>  -n <managedClusterView>
```
Comment (Anxhela21) [inline access_control/rbac_implement_rhacm.adoc]:
```suggestion
oc create rolebinding user-create-managedclusterview-binding --role=create-managedclusterview --user=<user>  -n <managedClusterName>
```

**Review by Anxhela21 — APPROVED** [review]
Just a minor correction to my comment before otherwise
/lgtm
/hold

**Comment by dockerymick** [comment]
thanks @oafischer and @Anxhela21 for the reviews, suggestions, and approvals. I have accepted the suggestions and ran the link check tool. Merging this PR

**Pull request merged**